Modules/Microsoft365DSC/DSCResources/MSFT_IntuneAppleMDMPushNotificationCertificate/settings.json

```diff
@@ -0,0 +1,44 @@
+﻿{
+    "resourceName": "IntuneAppleMDMPushCertificate",
+    "description": "Configures a resource for Apple MDM Push notification certificate used for device enrollment.",
+    "permissions": {
+        "graph": {
+            "delegated": {
+                "read": [
+                    {
+                        "name": "DeviceManagementManagedDevices.Read.All"
+                    },
+                    {
+                        "name": "DeviceManagementConfiguration.Read.All"
+                    }
+                ],
+                "update": [
+                    {
+                        "name": "DeviceManagementManagedDevices.ReadWrite.All"
+                    },
+                    {
+                        "name": "DeviceManagementConfiguration.ReadWrite.All"
+                    }
+                ]
+            },
+            "application": {
+                "read": [
+                    {
+                        "name": "DeviceManagementManagedDevices.Read.All"
+                    },
+                    {
+                        "name": "DeviceManagementConfiguration.Read.All"
+                    }
+                ],
+                "update": [
+                    {
+                        "name": "DeviceManagementManagedDevices.ReadWrite.All"
+                    },
+                    {
+                        "name": "DeviceManagementConfiguration.ReadWrite.All"
+                    }
```

Comment:
Permissions are duplicated.

Reply:
no, one is DeviceManagementManagedDevices and DeviceManagementConfiguration.

Reply:
Ahh, letter blindness strikes again. Sorry about that.